js/main.js

```diff
@@ -11,11 +20,21 @@ inputtag.addEventListener('input', (event) => {
 savebtn.addEventListener('click', () => {
   const iframe_src = `http://${ipaddress}:7011/ImageViewer?Mode=Motion&Resolution=320x240&Quality=Standard&Interval=10`
   cameraview.src = iframe_src
-  console.log(cameraview.src)
   // savebtn.textContent="NO";
 
   if(savebtn.textContent=="編集"){
-    savebtn.textContent="保存";}
+    savebtn.textContent="保存";
+    ptag.remove();
+    ptag.textContent = data;
```

Comment:
First, you removed the `ptag` 
Then set `textContent` to `ptag`

Do you think is it necessary to change the content which already we removed it?